### PR TITLE
feat: KPI-30 lambda set up to connect rds

### PR DESCRIPTION
**Pull request by kellycastrof**
## Jira Link

[Go to Jira](https://kpinetwork.atlassian.net/browse/KPI-XXX)

## Types of changes

- [ ] Hotfix (Bug detected)
- [x] Feature (Issue from an Active Sprint)

## Checklist:

- [x] Unit Test.
- [ ] Coverage.

### CODE REVIEW

**Review by herrera-luis — APPROVED** [review]

**Review by ridouku — APPROVED** [review]

File: .github/workflows/cd-kpinetwork-backend.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@ jobs:
           TF_VAR_aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
           TF_VAR_db_username: ${{ secrets.DB_USERNAME }}
           TF_VAR_db_password: ${{ secrets.DB_PASSWORD }}
+          TF_VAR_git_token: ${{ secrets.TOKEN }}
```
Comment (ridouku) [inline .github/workflows/cd-kpinetwork-backend.yml:61]: You should change the name, the name isn't enough descriptive (maybe CODEBUILD_TOKEN is a better name)

File: buildspec.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      python: 3.7
+  pre_build:
+    commands:
+      - pip3 install -r requirements/db.txt
+  build:
+    commands:
+      - cd db
+      - alembic upgrade head
+      - echo "FINISH"
+
```
Comment (ridouku) [inline buildspec.yml:29]: remove empty spaces

File: db/alembic.ini [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+# sys.path path, will be prepended to sys.path if present.
+# defaults to the current working directory.
+prepend_sys_path = .
+
+# timezone to use when rendering the date within the migration file
+# as well as the filename.
+# If specified, requires the python-dateutil library that can be
+# installed by adding `alembic[tz]` to the pip requirements
+# string value is passed to dateutil.tz.gettz()
```
Comment (ridouku) [inline db/alembic.ini:18]: I think is better remove all commented attributes

File: db/migrations/README [hunk 4]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (ridouku) [inline db/migrations/README:1]: If you want add a README.md, you should include more information, otherwise remove it

File: manifest.yml [hunk 5]
```diff
@@ -7,3 +7,11 @@ functions:
     include:
     - ./src/handlers/helloWorld/helloWorldHandler.py
     - ./src/service/helloWorld.py
+
+  dbSampleHandler:
+    # The dependencies of the helloWorldHandler function.
```
Comment (ridouku) [inline manifest.yml:12]: Remove this comments

**Review by ridouku — CHANGES_REQUESTED** [review]

File: infrastructure/buckets/s3.tf [hunk 6]
```diff
@@ -13,6 +13,13 @@ resource "aws_s3_bucket_object" "minimal_function_object" {
...
 
+resource "aws_s3_bucket_object" "db_sample_function_object" {
+  bucket = var.bucket_name
+  key = "dbSampleHandler.zip"
+  source = "${path.module}/../../dist/dbSampleHandler.zip"
+  etag = filemd5("${path.module}/../../dist/dbSampleHandler.zip")
+}
+
 # ----------------------------------------------------------------------------------------------------------------------
 # CREATE REFERENCE FROM S3 FILE UPLOADED
```
Comment (herrera-luis) [inline infrastructure/buckets/s3.tf:24]: Looks like there is a duplicated definition, the bucket name are the same **kpinetwork-backend** and **var.bucket_name**

Comment (kellycastrof) [reply infrastructure/buckets/s3.tf]: Done 👍

**Review by ridouku — APPROVED** [review]
Good job 👍

**Pull request merged**